topic-operator/src/main/java/io/strimzi/operator/topic/BatchingTopicController.java

```diff
@@ -1117,24 +1131,89 @@ private static Collection<AlterConfigOp> buildAlterConfigOps(Reconciliation reco
         return alterConfigOps;
     }
 
+    /**
+     * <p>The Topic Operator {@code alterableTopicConfig} can be used to specify a comma separated list of Kafka
+     * topic configurations that can be altered by users through {@code .spec.config}. Keep in mind that if changes
+     * are applied directly in Kafka, the operator will try to revert them producing a warning.</p>
+     *
+     * <p>This is useful in standalone mode when you have a Kafka service that restricts alter operations
+     * to a subset of all the Kafka topic configurations.</p>
+     *
+     * <p>The default value is "all", which means no restrictions in changing {@code .spec.config}.
+     * The opposite is "none", which can be set to explicitly disable any change.</p>
+     *
+     * @param alterConfigOps Requested alter config operations.
+     */
+    private void skipNonAlterableConfigs(Set<AlterConfigOp> alterConfigOps) {
+        var alterableConfigs = config.alterableTopicConfig();
```

Comment:
I am not a big fan of `var` ... it doesn't make it clear what type is the variable if the right hand side is not clear as well. For example, in this case, I had to navigate to the `alterableTopicConfig` to understand what's returning. Tbh I would stay with using explicit type declaration when there is no clarity.

Reply:
var is everywhere in the Topic Operator, so I would stick with it

Reply:
But while I can understand what it is when it's clear from the right hand side of the assignment, this case is not clear at all. If I don't have an IDE, I have to move across the code. Anyway if you decided to go with `var` (which I think is the worse thing ever added in Java) within TO I can just be accept it.